Packages/com.unity.inputsystem/InputSystem/Plugins/PlayerInput/PlayerInput.cs

```diff
@@ -1368,21 +1361,18 @@ private void InitializeActions()
             if (m_Actions == null)
                 return;
 
-            // Check if we need to duplicate our actions by looking at all other players. If any
-            // has the same actions, duplicate.
-            for (var i = 0; i < s_AllActivePlayersCount; ++i)
-                if (s_AllActivePlayers[i].m_Actions == m_Actions && s_AllActivePlayers[i] != this)
-                {
-                    var oldActions = m_Actions;
-                    m_Actions = Instantiate(m_Actions);
-                    for (var actionMap = 0; actionMap < oldActions.actionMaps.Count; actionMap++)
-                    {
-                        for (var binding = 0; binding < oldActions.actionMaps[actionMap].bindings.Count; binding++)
-                            m_Actions.actionMaps[actionMap].ApplyBindingOverride(binding, oldActions.actionMaps[actionMap].bindings[binding]);
-                    }
+            // duplicate action asset to not operate on the original(as it might be used outside - eg project wide action asset or UIInputModule)
```

Comment:
IMO this is much cleaner to not let it depend on some condition whether there is a clone or not.